src/framework/preferences/js/Panels.js

```diff
@@ -143,7 +133,8 @@ var fluid_2_0 = fluid_2_0 || {};
      * this is all done, the onDomBind event is fired.
      */
     fluid.prefs.subPanel.resetDomBinder = function (that) {
-        that.container = $(that.container.selector);
+        var userJQuery = that.container.constructor;
```

Comment:
Yes, the bug discovered was in the separated panel version of the UIO, dom binders of sub-panels were not reset due to the use of the incorrect jQuery instance. I'll add a TODO comment for this line.